lib/power_action_utils.pm

```diff
@@ -310,7 +310,7 @@ sub assert_shutdown_with_soft_timeout {
     my ($args) = @_;
     $args->{timeout}      //= 60;
     $args->{soft_timeout} //= 0;
-    $args->{bugref}       //= "No bugref specified";
+    $args->{bugref}       //= "bsc#1055462";
```

Comment:
I guess we shouldn't make the default value to be contained the bugref.

The soft-fail raised for that case, because the test suite contains LIVECD variable. And the appropriate issue is https://bugzilla.suse.com/show_bug.cgi?id=1096241.

So, my suggestion is to leave default value as is ('No bugref specified') and to just add 'bugref' to the soft_fail_data for LIVECD case:

```perl
    if (get_var('LIVECD')) {
        $soft_fail_data = {bugref=>'bsc#1096241', soft_timeout => 60, timeout => $shutdown_timeout *= 4};
    }
```

Actually it should already be specified. It seems like I missed this 'bugref' while implemented the subroutine.